addon/templates/components/polaris-checkbox.hbs

```diff
@@ -9,6 +9,7 @@
 }}
   <span class="Polaris-Checkbox {{if error "Polaris-Checkbox--error"}}">
     <input
+      data-test-polaris-checkbox-input
```

Comment:
to be consistent with other places....let's rename this to omit `polaris`, just `data-test-checkbox-input`